cilium-cli/connectivity/check/deployment.go

```diff
@@ -1812,6 +1821,32 @@ func (ct *ConnectivityTest) deployPerf(ctx context.Context) error {
 		return nil
 	}
 
+	if ct.params.PerfParameters.Bandwidth {
+		ct.params.PerfParameters.PodNet = false
+		ct.params.PerfParameters.HostNet = false
+		ct.params.PerfParameters.SameNode = false
+		ct.params.PerfParameters.OtherNode = false
+
+		var egressBandwidthAnnotations = annotations{egressBandwidth: "10M"}
+		var ingressBandwidthAnnotations = annotations{ingressBandwidth: "10M"}
+		ct.params.DeploymentAnnotations.Set(`{
+				"` + perClientEgressDeploymentName + `": ` + egressBandwidthAnnotations.String() + `,
+			    "` + perServerIngressDeploymentName + `": ` + ingressBandwidthAnnotations.String() + `
+			}`)
+		if err = ct.createServerPerfDeployment(ctx, perServerEgressDeploymentName, serverNode.Name, false); err != nil {
+			ct.Warnf("unable to create deployment: %w", err)
```

Comment:
nit: Might be good to qualify the error a bit more when this fails ie server/client name + direction